solvers/DPStokes/mobility.h

```diff
@@ -84,5 +164,1112 @@ class DPStokes: public libmobility::Mobility{
     Mobility::clean();
     dpstokes->clear();
   }
+
+  private:
+
+  double polyEval(std::vector<double> polyCoeffs, double x){
+
+    int order = polyCoeffs.size() - 1;
+    double accumulator = polyCoeffs[order];
+    double current_x = x;
+    for(int i = 1; i <= order; i++){
+      accumulator += polyCoeffs[order-i]*current_x;
+      current_x *= x;
+    }
+
+    return accumulator;
+  }
+
```

Comment:
All these functions should go outside of the class as free standing functions. I would even put them in another file, maybe `extra/dpstokes_parameters.h`.
If a method does not use the majority of the members in the class it should probably be a free-standing function.

Reply:
Thanks for the organizational tips! I moved all of these to a different file. I had linking/compilation problems, so I split it into a few files. @RaulPPelaez you can resolve this conversation if you think the solution is good enough (except for some documentation that I need to add), but let me know if you think I should adjust it further.